proxy/tcpproxy/userspace.go

```diff
@@ -36,12 +42,14 @@ func (r *remote) inactivate() {
 func (r *remote) tryReactivate() {
 	conn, err := net.Dial("tcp", r.addr)
 	if err != nil {
+		plog.Warningf("tryReactivate failed to connect to endpoint [%s] due to %v", r.addr, err)
```

Comment:
this will make the remote to stay inactive for another interval. we should also log that information.